index.html

```diff
@@ -1,21 +1,148 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Poppins:wght@600&display=swap"
+    rel="stylesheet" />
+  <title>Responsive grid project</title>
 </head>
 
 <body>
+  <!-- header -->
+  <header>
+    <nav>
+      <div class="box"></div>
+      <h1 class="title_top">
+        Maya Nelson </h1>
 
-    <!-- Add your markup here -->
+      <div class="ham_menu">
+        <div class="wrapper-menu">
+          <div class="line-menu half start"></div>
+          <div class="line-menu"></div>
+          <div class="line-menu half end"></div>
+        </div>
+      </div>
 
+      <p>PROJECT MANAGER</p>
+      <ul id="menu" class="menu">
+        <li><a href="index.html">ABOUT ME</a></li>
+        <li><a href="./css/resume.html">RESUME</a></li>
+        <li><a href="#">PROJECTS</a></li>
+        <li><a href="#">CONTACT</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <!------main----->
+  <main>
+  <div class="colored"></div>
+  <div class="mainDiv">
+    <div class="profile_box">
+      <img
+        src="https://static.wixstatic.com/media/2e2a49_bd3a1599bc1b4265a01b2754b99b8536~mv2_d_5760_3840_s_4_2.jpeg/v1/crop/x_2703,y_43,w_2199,h_2205/fill/w_205,h_205,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/AdobeStock_230407429.jpeg" />
+      <h2>
+        Maya
+        <br />
+        Nelson
+      </h2>
+      <div class="blue_line"></div>
+      <br>
+      <p> Project Manager</p>
+      <div class="social_div">
+        <a href="http://www.facebook.com">
+          <img alt="Facebook"
+            src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/0fdef751204647a3bbd7eaa2827ed4f9.webp" />
+        </a>
+        <a href="http://www.twitter.com">
+          <img alt="Twitter"
+            src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/c7d035ba85f6486680c2facedecdcf4d.webp" />
+        </a>
+        <a href="http://www.linkedin.com">
+          <img alt="LinkedIn"
+            src="https://static.wixstatic.com/media/6ea5b4a88f0b4f91945b40499aa0af00.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/6ea5b4a88f0b4f91945b40499aa0af00.webp" />
+        </a>
+        <a href="http://www.instagram.com">
+          <img alt="Instagram"
+            src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp" />
+        </a>
+      </div>
+    </div>
+    <div class="content">
+      <h1>Hello</h1>
+      <h4>Here's who I am & what I do</h4>
+      <div class="button_div">
+        <button>RESUME <a href="/resume.html" class="card-btn_1"></a></button>
+        <button>PROJECTS <a href="/projects.html" class="card-btn_2"></a></button>
```

Comment:
you should just use an `a` tag here rather than an `a` tag inside a button, `a` tags are used for navigation so this would be the best semantic html to use